docs/setup/new_environment.md

```diff
@@ -289,3 +289,9 @@ $ commcare-cloud monolith django-manage make_superuser {email}
 ```
 
 where `{email}` is the email address you would like to use as the username.
+
+## Step 13: [Optional] Add a new build
+
+To make a build of an application in CommCare, you will need to add a CommCare build to your server.
+
+To add a build to your server visit `/builds/edit_menu` and follow the instructions under "Import a new build from the build server"
```

Comment:
I went through your docs and didn't find any examples of you using a URL for their server. Wasn't sure if this notation would be better than `<url>/builds/edit_menu` or something else? Do you have any preference on how we link to self hosted pages?

Reply:
I think `<server url>` or something similar is fine.

Reply:
```suggestion
To add a build to your server visit `<server url>/builds/edit_menu` and follow the instructions under "Import a new build from the build server"
```